packages/common/src/configs.ts

```diff
@@ -90,6 +90,44 @@ export const envConfigs: Record<EnvironmentType, ProtocolConfig[]> = {
       configId: "local-31337-0",
       sellersBlackList: "",
       defaultDisputeResolverId: "1",
+      defaultTokens: [
```

Comment:
local blockchain does not have these tokens
I would suggest only native currency + 1 ERC20 token in "local": testErc20 (https://github.com/bosonprotocol/core-components/blob/87dd41e5d576be8f23a492bdcb46b72a2fd931d4/packages/common/src/configs.ts#L101C9-L101C18)